middlewares/authn.js

```diff
@@ -32,46 +35,42 @@ module.exports = function (req, res, next) {
 
   // look for load-testing override cookie
   if (req.cookies.load_test_token) {
-    if (
-      !csrf.checkToken(req.cookies.load_test_token, 'load_test', config.secretKey, {
-        maxAge: 24 * 60 * 60 * 1000,
-      })
-    ) {
+    const data = csrf.getCheckedData(req.cookies.load_test_token, config.secretKey, {
+      maxAge: 24 * 60 * 60 * 1000,
+    });
+
+    if (!data || !data.uuid || typeof data.uuid !== 'string' || !data.uuid.match(UUID_REGEXP)) {
```

Comment:
I opted not to retain backwards compatibility with the previous use case where we default to `loadtest@prairielearn.org`.